### PR TITLE
chore: add test data generator tool documentation

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

The Test Data Generator has been around for 1 year and more teams are using it. Adding it here will make it easier to discover for others. 

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
Hi @esthermmoturi, Can you please review it? I'm not sure if this is the best place for this tool documentation

File: content/en/core/overview/test-data-generator.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+title: "Test Data Generator"
+linkTitle: "Test Data Generator"
+weight: 2
+description: >
+  This tool is intended for application builders or CHT developer contributors that have knowledge on the design of docs from the CHT's CouchDB. Design the test data that fits your project hierarchy and reports. The tool will generate CouchDB docs and push them into your CHT test instance.
```
Comment (esthermmoturi) [inline content/en/core/overview/test-data-generator.md:6]:
```suggestion
  A tool for CHT developers to design, generate, and push CouchDB docs tailored to project hierarchies and reports .
```

**Comment by esthermmoturi** [comment]
Hey @latin-panda , I have reviewed and added a suggestion. We can leave the file where it's placed for now, I will move it later.

File: content/en/core/overview/test-data-generator.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+---
+
+# Test Data Generator for CHT Test Instances
+
+{{% alert title="Warning" %}} 
+It is not recommended to use this tool to push data into production instances.
+{{% /alert %}}
+
+This tool is intended for application builders or CHT developer contributors that have knowledge about the design of docs from the CHT's CouchDB.
```
Comment (esthermmoturi) [inline content/en/core/overview/test-data-generator.md:15]:
```suggestion
```

File: content/en/core/overview/test-data-generator.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+# Test Data Generator for CHT Test Instances
+
+{{% alert title="Warning" %}} 
+It is not recommended to use this tool to push data into production instances.
+{{% /alert %}}
+
+This tool is intended for application builders or CHT developer contributors that have knowledge about the design of docs from the CHT's CouchDB. 
+
+Design the test data that fit your project hierarchy and reports. The tool will generate CouchDB docs and push them into your CHT test instance.
```
Comment (esthermmoturi) [inline content/en/core/overview/test-data-generator.md:17]:
```suggestion
Design the test data that fits your project hierarchy and reports. The tool will generate CouchDB docs and push them into your CHT test instance.
```

**Review by esthermmoturi — CHANGES_REQUESTED** [review]

**Review by esthermmoturi — APPROVED** [review]
Thanks @latin-panda , approved!

**Pull request merged**